apps/wing/src/cli.ts

```diff
@@ -159,6 +159,7 @@ async function main() {
     )
     .option("-p, --plugins [plugin...]", "Compiler plugins")
     .option("-r, --rootId <rootId>", "App root id")
+    .option("-f, --test-filter <regex>", "Run tests that match the provided regex pattern")
```

Comment:
```suggestion
    .option("-f, --test-filter <regex>", "Run tests that match the provided regex pattern within the selected entrypoint files")
```

Reply:
(just because I find it confusing, since we can also run a few files at the same time)